dialog-el.html

```diff
@@ -296,7 +296,92 @@
       if (this.visible) this.close();
     };
 
-    proto.show = function() {
+    proto._resetPosition = function() {
+      this.style.top = 'initial';
+      this.style.right = 'initial';
+      this.style.bottom = 'initial';
+      this.style.left = 'initial';
+    }
+
+    proto.positionDialog = function(anchor) {
+      // If `anchor` is falsy OR the anchor and dialog don't share an
+      // offsetParent, early return
+      if (!anchor || anchor.offsetParent !== this.offsetParent) return;
```

Comment:
Alright, I have pulled in @liechtym's PR (#25) and have made some modifications to adjust for variable element `offsetParent`. This should remove the need for this assumption.

However this introduces a different assumption: that the dialog has the potential to flow **outside** of its container due to auto-positioning. I am going to look into how I could create a separate stacking context for dialogs (CSS fixed hack anyone?)